src/redux/actions/authActions.js

```diff
@@ -42,7 +43,7 @@ export const signInAccount = (userData, history) => async (dispatch) => {
     const { token } = logInUser.data.user;
     localStorage.setItem('jwtToken', token);
     const user = jwtDecode(token);
-    setAuthTokenHeader(token);
+    // setAuthTokenHeader(token);
```

Comment:
Take this statement and that on line 5 out if they are not needed anymore

Reply:
Okay

Reply:
done